src/analytics/analytics.service.ts

```diff
@@ -807,4 +809,24 @@ export class AnalyticsService {
 
     return result
   }
+
+  async getOnlineUserCount(pid: string) {
+    const keys = await redis.keys(`sd:*:${pid}`)
+
+    if (_isEmpty(keys)) {
+      return []
+    }
+
+    const sids = _map(keys, key => key.split(':')[1])
+
+    const query = `SELECT sid, dv, br, os, cc FROM analytics WHERE sid IN (${sids
+      .map(el => `'${el}'`)
+      .join(',')})`
+    const result = await clickhouse.query(query).toPromise()
+    const processed = _map(_uniqBy(result, 'sid'), el =>
+      _pick(el, ['dv', 'br', 'os', 'cc']),
+    )
+
+    return processed
+  }
```

Comment:
you can use this instead:
```suggestion
    return await redis.countKeysByPattern(`hb:${pid}:*`)
  }
```

Reply:
No, because this function does not exist.

Reply:
@rhaxma fyi it does https://github.com/Swetrix/swetrix-api/blob/main/src/common/constants.ts#L17

Reply:
fixed it at https://github.com/Swetrix/swetrix-api/pull/92/commits/ea583c23666972abdd39e64b5ffb15059f7318c9